README.md

```diff
@@ -1,193 +1,266 @@
-  augment Catalyst reaction network models with delays, and can simulate the
-  resulting stochastic chemical kinetics with delays models.
+ augment Catalyst reaction network models with delays, and can simulate the
+ resulting stochastic chemical kinetics with delays models.
 - [BondGraphs.jl](https://github.com/jedforrest/BondGraphs.jl) a package for
-  constructing and analyzing bond graphs models, which can take Catalyst models as input.
-- [PEtab.jl](https://github.com/sebapersson/PEtab.jl) a package that implements the PEtab format for fitting reaction network ODEs to data. Input can be provided either as SBML files or as Catalyst `ReactionSystem`s.
+ constructing and analyzing bond graphs models, which can take Catalyst models as input.
+- [PEtab.jl](https://github.com/sebapersson/PEtab.jl) a package that implements the PEtab format for 
+ fitting reaction network ODEs to data. Input can be provided either as SBML files or as Catalyst 
+ `ReactionSystem`s.
+
 
+## Illustrative example
 
-## Illustrative examples
-#### Gillespie simulations of Michaelis-Menten enzyme kinetics
+#### Deterministic ODE simulation of Michaelis-Menten enzyme kinetics
+Here we show a simple example where a model is created using the Catalyst DSL, and then simulated as 
+an ordinary differential equation.
 
 ```julia
-using Catalyst, Plots, JumpProcesses
-rs = @reaction_network begin
-  c1, S + E --> SE
-  c2, SE --> S + E
-  c3, SE --> P + E
+# Fetch required packages.
+using Catalyst, OrdinaryDiffEq, Plots
+
+# Create model.
+model = @reaction_network begin
+    kB, S + E --> SE
+    kD, SE --> S + E
+    kP, SE --> P + E
 end
-p  = (:c1 => 0.00166, :c2 => 0.0001, :c3 => 0.1)
-tspan = (0., 100.)
-u0 = [:S => 301, :E => 100, :SE => 0, :P => 0]
-
-# solve JumpProblem
-dprob = DiscreteProblem(rs, u0, tspan, p)
-jprob = JumpProblem(rs, dprob, Direct())
-jsol = solve(jprob, SSAStepper())
-plot(jsol; lw = 2, title = "Gillespie: Michaelis-Menten Enzyme Kinetics")
+
+# Create an ODE that can be simulated.
+u0 = [:S => 50, :E => 10, :SE => 0, :P => 0]
+tspan = (0., 200.)
+ps = (:kB => 0.01, :kD => 0.1, :kP => 0.1)
+ode = ODEProblem(model, u0, tspan, ps)
+
+# Simulate ODE and plot results.
+sol = solve(ode)
+plot(sol; lw = 5)
 ```
+![ODE simulation](docs/src/assets/readme_ode_plot.svg)
 
-![](https://user-images.githubusercontent.com/1814174/87864114-3bf9dd00-c932-11ea-83a0-58f38aee8bfb.png)
+#### Stochastic jump simulations
+The same model can be used as input to other types of simulations. E.g. here we instead perform a 
+jump simulation
+```julia
+# Create and simulate a jump process (here using Gillespie's direct algorithm).
+using JumpProcesses
+dprob = DiscreteProblem(model, u0, tspan, ps)
+jprob = JumpProblem(model, dprob, Direct())
+jump_sol = solve(jprob, SSAStepper())
+plot(jump_sol; lw = 2)
+```
+![Jump simulation](docs/src/assets/readme_jump_plot.svg)
 
-#### Adaptive time stepping SDEs for a birth-death process
 
+## Elaborate example
```

Comment:
Basically an example where I try to show off as much as possible while still keeping things simple/short.